JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -29,12 +34,34 @@ module Microsoft.ApplicationInsights.Context {
          * The store region.
          */
         public storeRegion: string;
-        
+
+        static cookieSeparator: string = '|';
+
+        public setAuthId(id: string, accountId? : string) {
+            this.authId = id;
+            var authCookie = this.authId;
+
+            if (accountId) {
+                this.accountId = accountId;
+                authCookie = [this.authId, this.accountId].join(User.cookieSeparator);
+            }
+            
+            // Set the suth id cookie. No expiration date because this is a session cookie (expires when browser closed).
+            Util.setCookie('ai_authUser', authCookie);
```

Comment:
Consider moving the `ai_authUser` to a `static` field like the cookie separator since this string is used at multiple places.